datastore/gcloud/aio/datastore/constants.py

```diff
@@ -81,4 +83,7 @@ class TypeName(enum.Enum):
     LatLng: TypeName.GEOPOINT,
     str: TypeName.STRING,
     type(None): TypeName.NULL,
+    # We should use .entity.Entity here, but that would lead to a lot of
+    # unsolveable import cycles.
+    Any: TypeName.ENTITY,
```

Comment:
Hmm... I see the problem here, but I'm a bit worried about adding `Any` here as a key, as a few bad type hints could lead us to mapping something as an `ENTITY` incorrectly down the line. Instead of adding anything to this `TYPES` dictionary, you could solve this issue by adding it as a late-binding in `Value._get_supported_types()`, eg. by doing:
```python
    @classmethod
    def _get_supported_types(cls):
        supported_types = TYPES
        supported_types.update({
            cls.key_kind: TypeName.KEY,
+           Entity: TypeName.ENTITY,
        })
        return supported_types
```

Reply:
Hi, thanks for your thoughtful comments, I've adjusted this per your suggestion.